examples/s3-import-mysql/main.tf

```diff
@@ -80,17 +80,25 @@ module "security_group" {
   tags = local.tags
 }
 
-module "import_s3_bucket" {
-  source  = "terraform-aws-modules/s3-bucket/aws"
-  version = "~> 2.0"
-
+# Temporary work around until S3 module is updated to support v4.x
```

Comment:
FYI I am planning to work on the S3 bucket module on Friday and probably next week.